src/core.ts

```diff
@@ -202,6 +202,16 @@ interface SiteAdapterOptions {
       (Enables special handling of certain HTML elements, which doesn't
        work well on all sites) */
   iframe?: boolean;
+
+  /**
+   * Styling function for the selected row
+   */
+  styleSelectedRow?(row: DataRow): void;
```

Comment:
Thoughts on passing in the list of elements here, rather than the DataRow?

Also maybe `onRowSelected` would be a more generic way to describe this function, as a "hook" that lets you add arbitrary behavior when a row is selected -- thoughts?

Reply:
I switched it to passing in the list of elements since I realized I was doing that anyways with the unselected rows function. I also updated the selected row function to be onRowSelected as described. Who knows what we might want to do in the future, so making it a function that is more broad and flexible makes sense to me.